### PR TITLE
Added Graphite to Vagrant image

### LINKED ISSUE #325: Add Graphite to Vagrant image
Add Graphite to default Vagrant image.

### PR DESCRIPTION
**Pull request by ingwarsw**
- Closes #325

### CODE REVIEW

**Comment by chemicL** [comment]
lgtm :+1:

File: vagrant_provisioning/conf/hermes-console-config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "title": "hermes vagrant console"
   },
   "dashboard": {
-    "metrics": "",
+    "metrics": "http://PUBLIC_IP:8082",
```
Comment (dankraw) [inline vagrant_provisioning/conf/hermes-console-config.json]: why not 10.10.10.10? :>

Comment (ingwarsw) [inline vagrant_provisioning/conf/hermes-console-config.json]: Because if someone changes IP in Vagrantfile will not have to do same in X places.

Comment (dankraw) [inline vagrant_provisioning/conf/hermes-console-config.json]: ok, I see it's replaced in the shell script.

**Comment by dankraw** [comment]
:+1:

**Pull request merged**